styles.css

```diff
@@ -112,15 +70,55 @@ img {
   margin-bottom: auto;
 }
 
+a {
+  color: #ffffff;
+
+  text-decoration: none;
+}
+
+.type-of-message {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 38vw;
+  height: 9vh;
+  padding: 1rem;
+  margin-top: .6rem;
+
+  color: #000000;
+
+  font-style: italic;
+}
+
+.radio-buttons {
+  display: flex;
+  justify-content: space-around;
+  width: 38vw;
+  margin: .25rem .5rem 1rem .5rem;
+
+}
+
+.message {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 60vw;
+  height: 20vh;
+}
+
+.random-message {
+  margin: auto;
+
+  color: #000000;
+
+  font-family: 'Quicksand', sans-serif;
```

Comment:
I think since you have this outlined in `body` global styling, you don't need font-family styling here!